SofaKernel/modules/SofaDeformable/src/SofaDeformable/StiffSpringForceField.inl

```diff
@@ -36,6 +36,12 @@ template<class DataTypes>
 StiffSpringForceField<DataTypes>::StiffSpringForceField(double ks, double kd)
     : StiffSpringForceField<DataTypes>(nullptr, nullptr, ks, kd)
 {
+    this->addUpdateCallback("updateSprings-ks-kd", { &d_indices1, &d_indices2, &d_length, &this->ks, &this->kd}, [this](const core::DataTracker& t)
+    {
+        SOFA_UNUSED(t);
+        createSpringsFromInputs();
+        return sofa::core::objectmodel::ComponentState::Valid;
+    }, {&this->springs});
```

Comment:
First constructor is calling the second one. So I'm not 100% sure but you can remove this one otherwise your callback will be register twice.